--- build_steps.sh	2021-02-09 09:01:08.000000000 -0800
+++ build_steps.sh.new	2021-02-09 09:03:41.000000000 -0800
@@ -12,24 +12,23 @@
 export CI_SUPPORT="${FEEDSTOCK_ROOT}/.ci_support"
 export CONFIG_FILE="${CI_SUPPORT}/${CONFIG}.yaml"
 
-cat >~/.condarc <<CONDARC
-
-conda-build:
- root-dir: ${FEEDSTOCK_ROOT}/build_artifacts
-
-CONDARC
+# cat >~/.condarc <<CONDARC
+# 
+# conda-build:
+#  root-dir: ${FEEDSTOCK_ROOT}/build_artifacts
+# 
+# CONDARC
 
 conda install --yes --quiet "conda-forge-ci-setup=3" conda-build pip -c conda-forge
 
 # set up the condarc
 setup_conda_rc "${FEEDSTOCK_ROOT}" "${RECIPE_ROOT}" "${CONFIG_FILE}"
 
-source run_conda_forge_build_setup
+# source run_conda_forge_build_setup
 
 # make the build number clobber
 make_build_number "${FEEDSTOCK_ROOT}" "${RECIPE_ROOT}" "${CONFIG_FILE}"
 
-
 if [[ "${BUILD_WITH_CONDA_DEBUG:-0}" == 1 ]]; then
     if [[ "x${BUILD_OUTPUT_ID:-}" != "x" ]]; then
         EXTRA_CB_OPTIONS="${EXTRA_CB_OPTIONS:-} --output-id ${BUILD_OUTPUT_ID}"
@@ -50,4 +49,4 @@
     fi
 fi
 
-touch "${FEEDSTOCK_ROOT}/build_artifacts/conda-forge-build-done-${CONFIG}"
\ No newline at end of file
+# touch "${FEEDSTOCK_ROOT}/build_artifacts/conda-forge-build-done-${CONFIG}"
